lib/elixir/pages/getting-started/basic-types.md

```diff
@@ -322,13 +322,22 @@ iex> 1 == 2.0
 false
 ```
 
-However, you can use the strict comparison operator [`===`](`===/2`) and [`!==`](`!==/2`) if you want to distinguish between integers and floats (that's the only difference between these operators):
+However, you can use the strict comparison operator [`===`](`===/2`) and [`!==`](`!==/2`) if you want to distinguish between integers and floats:
 
 ```elixir
 iex> 1 === 1.0
 false
 ```
 
+Besides this distinction, since Erlang 27, strict comparison operators are also considering `+0.0` and `-0.0` as non-equal:
+
+```elixir
+iex> +0.0 === -0.0
+false
+iex> +0.0 == -0.0
+true
+```
+
```

Comment:
I am not sure I want to go into this level of distinction for a getting started guide. I would just remove the note and leave the content as it was, and we can mention the differences in the `===` and `!==` docs.

In particular, IEEE recommends `+0.0` and `-0.0` to actually be equal. But `===` is about "are they the same term"?